.github/workflows/velox.yml

```diff
@@ -0,0 +1,76 @@
+name: Velox Nightly Build and Test
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  container:
+    runs-on: ubuntu-latest
+    container: prestocpp/velox-velox-torcharrow:kpai-20220301
+    steps:
+      - name: Print CPU info
+        run: cat /proc/cpuinfo
+
+      - name: Setup Python environment
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Check out source repository
+        uses: actions/checkout@v2
+        with:
+            submodules: true
+
+      - name: Update submodules
+        run: |
+          git submodule update --init --recursive
+          cd csrc/velox/velox/
+          git submodule update --remote --merge
+
+      - name: Setup conda environment
+        run: | 
+          source /opt/conda/etc/profile.d/conda.sh
+          conda create --name ta_python37 python=3.7 -y
+          echo "/opt/conda/condabin" >> $GITHUB_PATH
+          echo "/opt/conda/envs/ta_python37/bin" >> $GITHUB_PATH
+
+      - name: Bulid TorchArrow
+        run: |
+          python3 setup.py develop --user
+
+      - name: Install dependencies for test
+        run: |
+          pip3 install pytest  --user
+          pip3 install torch==1.10.0+cpu -f https://download.pytorch.org/whl/cpu/torch_stable.html --user
+
+      - name : Run TorchArrow unit tests
+        run: |
+          export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/lib/
+          python3 -m pytest --no-header -v torcharrow/test
```

Comment:
there is chance for the test to fail with "illegal instructions", when GitHub Actions runs on a `Intel E5-2673 v3/v4` : https://github.com/facebookresearch/torcharrow/pull/92#issuecomment-982850395

Reply:
So  are you saying we dont build torch arrow unit tests etc, and just go directly to building wheel ?

Reply:
Correct.